go.mod

```diff
@@ -5,11 +5,13 @@ go 1.17
 require (
 	cloud.google.com/go/bigtable v1.13.0
 	github.com/google/go-cmp v0.5.7
+	github.com/jhump/protoreflect v1.10.3
 	golang.org/x/oauth2 v0.0.0-20211104180415-d3ed0bb246c8
 	golang.org/x/sys v0.0.0-20220207234003-57398862261d
 	google.golang.org/api v0.67.0
 	google.golang.org/grpc v1.44.0
 	google.golang.org/protobuf v1.27.1
+	gopkg.in/yaml.v2 v2.2.3
```

Comment:
It's actually used as a guide for formatting output. I'm okay with this dependency for now.

Reply:
After chatting, it sounds like we're okay with this dependency (for now), given that this code is primarily distributed as an executable. I'll add notes in the go.mod file that mention that we have some reservations about this library (and the proto) library.

Reply:
I think since we are committed to at least the YAML dependency indefinitely, no need to have notes there. I would maybe add a note to the repo's README.md that this library includes non-google dependencies.